integration_testing/features/android/admin-library-page.feature

```diff
@@ -0,0 +1,19 @@
+Feature: My downloads - Library page
+
+  Background:
+    Given I am signed in as a super admin user
+    	And I am connected to the Internet
+			And I am at *Learn > Library*
+			And there are imported channels with resources on the device
+
+	Scenario: Super admin is able to see Kolibri Studio libraries
+		When I load the *Learn > Library* page
+			And I look at the *Other libraries* section of the page
+		Then I see the *Kolibri content library* section
+			And I see up to 5 chards on up to 2 rows
```

Comment:
Typo, should be ``cards`` (extra ``h``).
``Kolibri Content Library`` should be capitalized title case, it's a product name.